cloudinit/sources/DataSourceOVF.py

```diff
@@ -684,4 +740,69 @@ def _raise_error_status(prefix, error, event, config_file, conf):
     util.del_dir(os.path.dirname(config_file))
     raise error
 
+
+def load_cloudinit_data(md_path, ud_path):
+    """
+    Load the cloud-init meta data, user data, cfg and network from the
+    given files
+    """
+    LOG.debug('load meta data from: %s: user data from: %s',
+              md_path, ud_path)
+    md = {}
+    cfg = {}
```

Comment:
Don't need this local var, just return {} as the 3rd element in the return tuple